packages/dd-trace/test/profiling/profiler.spec.js

```diff
@@ -76,246 +76,308 @@ describe('profiler', function () {
     profilers = [wallProfiler, spaceProfiler]
 
     sourceMapCreate = sinon.stub()
+  }
 
-    Profiler = proxyquire('../src/profiling/profiler', {
-      '@datadog/pprof': {
-        SourceMapper: {
-          create: sourceMapCreate
+  describe('not serverless', function () {
```

Comment:
Can we please avoid changing so much indentation unnecessarily? It makes it a lot harder to review and makes git blame unhelpful.

You could move this into a function outside of the outer describe and then use it like `describe('not serverless', notServerlessTests.bind(null, a, b, c))` with the a, b, c bit to pass in anything you need to get from this scope. I generally try as much as possible to avoid significant diff changes when there's no major logical changes to warrant it.

Reply:
I personally prefer changing indentation and avoid unnecessary abstractions instead of trying to work around this with code. You can always add `?w=1` in the URL to remove the whitespaces from the diff (not sure if that can be done with the UI, couldn't find it).